content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -155,145 +124,114 @@ enum Level {
 
-## 6. Import your Project into the Prisma Data Platform
-
-With Cloudflare Workers, you can't directly access your database because there is no TCP support. Fortunately, Prisma has your back with the [Data Proxy](/data-platform/classic-projects/data-proxy).
-
-1. To get started, sign up for a free [Prisma Data Platform account](https://cloud.prisma.io/).
-
-   <Admonition type="info">
-
-   **Note**<br /><br />
-   A GitHub account is required to sign up for the Prisma Data Platform.
-
-   </Admonition>
-
-   ![Signup for Prisma Data Platform](../../../doc-images/data-platform/pdp-signup.png)
+## 4. Enable Accelerate in Prisma Data Platform
 
-2. Click **New Project** and then select **Import a Prisma repository**.
-3. Fill in the repository and project details, and click **Next**.
+With Cloudflare Workers, you can't directly access your database because there is no TCP support. Fortunately, Prisma has your back with [Prisma Accelerate](/data-platform/accelerate).
 
-   ![Import a Project](./images/450-03-import-project.png)
+To get started, sign up for a free [Prisma Data Platform account](https://console.prisma.io/), enable Accelerate and generate an Accelerate connection string.
 
-4. Next, you connect the Prisma Data Platform to the MongoDB Atlas database and set up the Data Proxy.
+## 5. Set the Accelerate connection string in your environment
 
-   1. Paste the MongoDB Atlas connection string which also includes the database name as described in [1. Get a MongoDB Atlas connection string](#1-get-a-mongodb-atlas-connection-string).
-   2. Under **Location**, select a Data Proxy location that is geographically close to your MongoDB database location.
-   3. Enable **Static IPs** and copy and paste them in the _Network Access_ screen in MongoDB Atlas.
-
-      ![Connect your Database](./images/450-04-connect-db.png)
-
-   4. Click **Create Project** to test the connection and set up the Data Proxy.
-
-      <Admonition type="warning">
-
-      If you see, "The database needs to be empty to proceed", you can simply use a different database name. Using the screenshot above, that would be renaming `Log` to something else.
-
-      </Admonition>
-
-      If all goes well, you'll be greeted with a new connection string that starts with `prisma://`.
-
-   5. Copy the `prisma://` connection string to your clipboard.
-
-      ![Data Proxy Page](./images/450-05-data-proxy.png)
-
-## 7. Set the Data Proxy Connection string in your environment
-
-1. Add the Data Proxy connection string to your local environment `.env` file.
+1. Add the Prisma Accelerate connection string to your local environment `.env` file.
 
    ```diff file=.env
-   - DATABASE_URL = "mongodb+srv://<user>:<password>@cluster.mongodb.net/Log?retryWrites=true&w=majority"
-   + DATABASE_URL = "prisma://aws-us-east-1.prisma-data.com/?api_key=•••••••••••••••••"
+   +DATABASE_URL="DATABASE_URL="prisma://accelerate.prisma-data.net/?api_key=__API_KEY__"
```

Comment:
Nice how you run into the same problem that many our users will hit :D 

```suggestion
   +DATABASE_URL="prisma://accelerate.prisma-data.net/?api_key=__API_KEY__"
```

Reply:
Incredible catch, thanks 🙏 !